classes/classes_hw.py

```diff
@@ -0,0 +1,142 @@
+import matplotlib.pyplot as plt
+from Bio import Seq
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+
+
+# task 1
+class BioinformaticsStudent:
+    def __init__(self, name, course, city, skills=None):
+        self.skills = skills
+        if self.skills is None:
+            self.skills = set()
+        self.name = name
+        self.course = course
+        self.city = city
+
+    def subjects_semester1(self):
+        bio_subjects = ['Discrete math,',
+                        'Statistics and R,',
+                        'Python programming,',
+                        'Command line,',
+                        'Bioinformatics practice']
+        programmer_subjects = ['Algorithms,',
+                               'Statistics,',
+                               'Molecular biology,',
+                               'Bioinformatics practice']
+        if self.course == 'Bioinformatics for biologists':
+            print(*bio_subjects)
+        elif self.course == 'Algorithmic bioinformatics':
+            print(*programmer_subjects)
+
+    def subjects_semester2(self):
+        bio_subjects = ['Introduction to ML,',
+                        'Python programming,',
+                        'Jornal club,',
+                        'Molecular phylogenetics and evolution,',
+                        'Bioinformatics practice,',
+                        'Project']
+        programmer_subjects = ['ML,',
+                               'Statistics,',
+                               'Jornal club,',
+                               'Molecular phylogenetics and evolution,',
+                               'Bioinformatics practice,',
+                               'Project']
+        if self.course == 'Bioinformatics for biologists':
+            print(*bio_subjects)
+        elif self.course == 'Algorithmic bioinformatics':
+            print(*programmer_subjects)
+
+    def learning(self, new_skill):
+        self.skills.add(new_skill)
+
+    def meeting(self, student):
+        if self.city == student.city:
+            print(f'{self.name} and {student.name} can meet offline in {self.city}!')
+        else:
+            print(f'{self.name} and {student.name} can meet only online')
+
+    def skills_show(self):
+        print(f'{self.name} can', *self.skills)
+
+
+# task 2
+class Rna:
+    def __init__(self, sequence):
+        self.sequence = Seq.Seq(sequence)
+
+    def translation(self):
+        translated_sequence = self.sequence.translate()
+        return translated_sequence
+
+    def reverse_transcription(self):
+        dna = self.sequence.back_transcribe()
+        return dna
+
+
+# task 3
+class PositiveSet(set):
+    def __init__(self, *args):
+        if len(args) == 0:
+            super().__init__(*args)
+        else:
+            for i in args:
+                if i > 0:
+                    super(PositiveSet, self).add(i)
```

Comment:
Так делали во втором питоне. В 2022 году можно писать просто
```suggestion
                    super().add(i)
```